packages/server/graphql/mutations/dragEstimatingTask.ts

```diff
@@ -48,14 +45,19 @@ export default {
       return standardError(new Error('Meeting phase not found'), {userId: viewerId})
     }
     const {stages} = estimatePhase
-    const draggedStage = stages.find((stage) => stage.id === stageId)
-    if (!draggedStage) {
-      return standardError(new Error('Meeting stage not found'), {userId: viewerId})
+    const draggedStages = stages.filter((stage) => stage.taskId === taskId)
+    if (!draggedStages.length) {
+      return standardError(new Error('No meeting stages were found'), {userId: viewerId})
     }
+    const stageIds = draggedStages.map((stage) => stage.id)
 
     // RESOLUTION
     // MUTATIVE
-    draggedStage.sortOrder = sortOrder
+    const noise = Math.random() / 1e10
```

Comment:
We need a small positive number as increment to preserve the dimension ordering. The `dndNoise()` produces both negative and positive numbers.

Reply:
-1 this is an interesting constraint!
dndNoise is small enough to let someone drop 1 item between 2 items up to ~32 times before it got too small, which was probably fine.
with a larger noise coefficient + multiple items safely reorder `Math.floor(10 / dimensionCount)` times. That might not be enough for real world usage. I'll check my math & try to break it in testing. We may just have to update every succeeding stage sortOrder. I don't love the O(log(n)) updates, but better than running into that really weird bug again

Reply:
I knew this will get your attention! What I found out before is that sometimes the number generated from `dndNoise` with `Math.random() / 1e15` is too small it's considered equivalently as zero. A series of output from my experimental tests:

<img width="433" alt="Screen Shot 2022-02-18 at 11 43 10 AM" src="https://user-images.githubusercontent.com/1879975/154751104-0a185f56-606e-4735-b438-5fa9162b92a0.png">

Reply:
interesting! that's a good enough reason to move away from a float sort.
what do you think about moving to an integer-based sort? 
discussion here: https://github.com/ParabolInc/parabol/issues/6086